letsencrypt/DOCS.md

```diff
@@ -513,39 +532,68 @@ on the DNS zone to be used for authentication.
 
 </details>
 
-## Certificate files
+## Certificate Files
 
-The certificate files will be available within the "ssl" share after successful request of the certificates.
+The certificate files will be available within the `/ssl` share after successful request of the certificates. By default other addons are referring to the correct path of the certificates. Note that the `Let's Encrypt` log files may tell you that the certificate files are located under `/data/letsencrypt/live/` but that is NOT true. They are under the `/ssl` directory.
 
-By default other addons are referring to the correct path of the certificates.
-You can in addition find the files via the "samba" addon within the "ssl" share.
+After clicking **Save** on the configuration, go back to the **Info** tab and click **Start**. That will engage Certbot and start requesting the certificate. You can view the status under the **Log** tab.
 
-## Supported DNS providers
+## Configuration
+
+Once the certificates are created, you will need to adjust the HTTP section of your `configuration.yaml` file. The easiest way to adjust the configuration file is by installing the **Studio Code Server** add-on. Then you can just click the button on the left side of Home Assistant, make your changes, press Ctrl+S, and you're done!
+
+Here is the minimum configuration required in order for SSL certificates to work properly:
 
-```txt
-dns-azure
-dns-cloudflare
-dns-cloudxns
-dns-digitalocean
-dns-directadmin
-dns-dnsimple
-dns-dnsmadeeasy
-dns-gehirn
-dns-google
-dns-hetzner
-dns-linode
-dns-luadns
-dns-njalla
-dns-nsone
-dns-ovh
-dns-rfc2136
-dns-route53
-dns-sakuracloud
-dns-netcup
-dns-gandi
-dns-transip
-dns-inwx
 ```
+http:
+  ssl_certificate: /ssl/fullchain.pem
+  ssl_key: /ssl/privkey.pem
+```
+
+You may want to change additional settings, but the two above are required for SSL certificates to work correctly.
+
+[Full HTTP section documentation](https://www.home-assistant.io/integrations/http/)
```

Comment:
If you want to go into this part here then we should do the recommended way. Adding your SSL certificate to the HA configuration is not recommended outside of a core or container install (since those have no other options). If addons are available the recommended approach is to install the [nginx proxy add-on](https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_nginx_proxy) and add your SSL certificate there.

The problem with adding the certificates to HA directly is you make LAN access difficult or impossible for most users. HA can now only be accessed via SSL even on LAN, there is no HTTP-only option. And that means if the user tries to go to `homeassistant.local:8123` (or whatever they use for their LAN URL) they get certificate errors that they now have to figure out how to deal with and their mobile app no longer works on LAN.

So if we want to recommend how they use these options in conjunction with HA we need to describe the recommended approach. Use a reverse proxy (the nginx proxy addon is by far the simplest), do not add them directly to HA config.

Reply:
Hey sorry for the delay I got sick with Covid 😷 but am better now.

That makes sense.

So what should I do instead since this is for the Let's Encrypt doc? Is there any other reason to use LE other than to get your own SSL cert?